src/api/ContentfulPreviewLink.js

```diff
@@ -0,0 +1,18 @@
+import { ApolloLink } from 'apollo-link';
+
+export default ({ cookieStore }) => {
+	return new ApolloLink((operation, forward) => {
+		// Only do this on contentful queries
+		if (operation.operationName.includes('contentful')) {
+			// Fetch current basket id from cookie
```

Comment:
basket id?

Reply:
> basket id?

... Is the  apollo link I copied and pasted 😆